test/libsolidity/syntaxTests/constantEvaluator/type_reference.sol

```diff
@@ -0,0 +1,3 @@
+int[L] constant L = 6;
```

Comment:
Is this syntactically valid in a `contract` context too? If so, can add another test case for that?